sql/core/src/test/resources/test-data/value-malformed-with-header.csv

```diff
@@ -0,0 +1,3 @@
+a,b
+0,2013-111-11 12:13:14
```

Comment:
Is the value wrong because of wrong month (111) or wrong type (timestamp)? Can we make the example smaller.